packages/deno/test/transport.ts

```diff
@@ -0,0 +1,30 @@
+import { createTransport } from 'npm:@sentry/core';
+import type {
+  BaseTransportOptions,
+  Envelope,
+  Transport,
+  TransportMakeRequestResponse,
+  TransportRequest,
+} from 'npm:@sentry/types';
+import { parseEnvelope } from 'npm:@sentry/utils';
```

Comment:
h: Let's ditch all of the `npm:` prefixes. Rollup will eliminate imports through bundling anyhow.

Reply:
This is in the tests which are run through Deno. Unfortunately Deno doesnt support loading any of the local workspace dependencies! 
https://discord.com/channels/684898665143206084/684911491035430919/1161366243081736322

Reply:
Ah alright. This makes sense! Thanks for educating me Disregard my comment then. :D